docker/docker.settings.php

```diff
@@ -67,3 +67,29 @@
 //   $conf['memcache_bins'] = array(
 //     'cache' => 'default',
 //   );
+
+
+// Configure memcache.
```

Comment:
Based on this can we then remove the "Memcache setup" from the TODO list in the README?

Reply:
Good point - i've removed it form the readme